src/components/common/SystemCommands.vue

```diff
@@ -55,40 +55,47 @@
 
     <v-list-group
       prepend-icon="$restart"
-      no-action>
```

Comment:
@randellhodges I just noticed you removed `no-action` from here, I assume this was to get the text to fully align to the left?

That change however, makes it look rather odd when we compare with the items under "Host" and "Power" entries as they still show the margin on the left, so I wonder if making the sidebar width going to 300px wouldn't be a better option:

![image](https://user-images.githubusercontent.com/85504/154981149-f5f7470a-8783-479c-8e69-d0475dd21bbc.png)

Reply:
I went back and forth with that. I didn't consider making the sidebar wider. I think that would indeed look better. I was attempting to prevent `moonraker-telegram-bot` from wrapping. I know we won't be able to prevent every name from ever wrapping, but I think your idea of making it a little wider is a better compromise.

Reply:
Doing it! 🙂

Reply:
Done!